do-dont/src/InstructionsCard.ts

```diff
@@ -20,29 +17,24 @@ export abstract class InstructionsCard extends LitElement {
   @property()
   label: string;
 
-  render() {
-    const icon =
-      typeof this.icon === 'function' ? this.icon(this.color) : this.icon;
+  instanceStyle: CSSResult;
 
+  render() {
     return html`
-      <div class="container">
-        <div class="component-container">
+      <style>
+        ${this.instanceStyle}
+      </style>
+      <div part="container" class="container">
+        <div part="component-container" class="component-container">
           <slot name="component"></slot>
         </div>
-        <div
-          class="instructions-container"
-          style="
-            border-top-color: ${this.color};
-          "
-        >
-          <div class="background" style="background-color: ${this.color}"></div>
+        <div part="instructions-container" class="instructions-container">
+          <div class="background"></div>
           <div class="title-container">
-            ${icon}
+            ${this.icon}
```

Comment:
As discussed this rely on the fact that `icon` is a Lit template.
Maybe there is a way to let the icon be styled by CSS background instead to style do and dont

Reply:
Or a slot

Or not configurable at all in a just version

Reply:
icon is defined as `string | TemplateResult`, so string was assumed, but it won't work due to XSS protection, or `unsafeHTML` is necessary, which is also not good
slot is best solution indeed, with default slot content in Do and Dont respectively